lessons/02-javascript/INCLASS.md

```diff
@@ -0,0 +1,34 @@
+# Lesson 2 In-Class Exercises
+(As before, we’re not expecting you to finish all of these activities! Pick the ones which are interesting and challenging to you, and feel free to work with the people around you.)
+
+## JavaScript Concepts:
+- Break into groups of ~4 members. Each member is in charge of finding out about a specific javascript concept and writing a small jsfiddle that demonstrates it
+  - Comeback together and give quick whiteboard talks about your concepts
+  - Concepts:
+    - First-Class and Higher Order Functions
+    - Prototypes/Inheritance
+	- Bind, Apply, Call
+    - Closures
+
+## Event Listeners:
+- Much of Web development is learning to react to event, as first foray into this write a small codepen [like this](http://codepen.io/jagreene/pen/pjMRvb) that creates a simple html element and responds to an event of your choice.
```

Comment:
react to eventS
first foray into Event Handling
(perhaps list some events that might be interesting to play with incase people don't think of anything other than click. Eg. scroll, keydown, mouseenter (https://developer.mozilla.org/en-US/docs/Web/Events))